deploy/docker-compose.yml

```diff
@@ -15,11 +15,13 @@ services:
     volumes:
       - ./nginx/default.conf.template:/etc/nginx/templates/default.conf.template:ro
     environment:
-      - S3_PUBLIC_URL=${S3_PUBLIC_URL:-http://localhost/s3/}
-      - S3_LOCAL_URL=${S3_lOCAL_URL:-http://minio:9000}
+      - FUZON_PUBLIC_URL=${FUZON_PUBLIC_URL:-http://localhost/fuzon}
       - HTSGET_PUBLIC_URL=${HTSGET_PUBLIC_URL:-http://localhost/htsget}
+      - S3_PUBLIC_URL=${S3_PUBLIC_URL:-http://localhost/s3/}
+      - FUZON_LOCAL_URL=${FUZON_LOCAL_URL:-http://fuzon:8080}
       - HTSGET_LOCAL_URL=${HTSGET_LOCAL_URL:-http://htsget:8080}
       - MODOS_LOCAL_URL=${MODOS_LOCAL_URL:-http://modos-server:8000}
+      - S3_LOCAL_URL=${S3_lOCAL_URL:-http://minio:9000}
```

Comment:
Call me a hawk
```suggestion
      - S3_LOCAL_URL=${S3_LOCAL_URL:-http://minio:9000}
```

Reply:
daaamn good catch, I wish I had a linter for this kind of stuff :thinking: